src/lms-ds-loader/lms_ds_loader/mssql_lms_operations.py

```diff
@@ -0,0 +1,196 @@
+# SPDX-License-Identifier: Apache-2.0

```

Comment:
Could this be simplified by dropping the dataclass and just having functions in the module?

Reply:
There are other classes where that might work, but for this one no: the class is instantiated early on, after reading the CLI input arguments, and passed from function to function. In this way we can pass around either an MSSQL object or a PostgreSQL object.